contrib/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ add_subdirectory (murmurhash)
 add_subdirectory (replxx-cmake)
 add_subdirectory (ryu-cmake)
 add_subdirectory (unixodbc-cmake)
+add_subdirectory (xz)
```

Comment:
Looks like you've forget to add submodule

Reply:
Isn't it?
https://github.com/ClickHouse/ClickHouse/pull/16578/files#diff-fe7afb5c9c916e521401d3fcfb4277d5071798c3baf83baf11d6071742823584R193

Reply:
Not really, this is just to know where to fetch the submodule from.

You need to run smth like this:

```
git submodule add https://github.com/xz-mirror/xz contrib/xz
```

And this will create a special file `contrib/xz` in the git index (but in the filesystem you will see it as a content of the repository), that will contain the HEAD for the submodule

Reply:
Also to pass fasttest you need to add `contrib/xz` into https://github.com/ClickHouse/ClickHouse/blob/f10a5207f4e89f73779375a042185b64b0b8db39/docker/test/fasttest/run.sh#L130

Reply:
Also it's possible to disable `xz` in fasttest.